sdk/react/panels/cast/CastDevicesScreen.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Animated,
+  ListView,
+  Text,
+  View,
+  TouchableHighlight,
+} from 'react-native';
+
+import {
+  BUTTON_NAMES,
+} from '../../constants';
+import Utils from '../../utils';
+
+import castScreenStyles from '../style/castScreenStyles.json';
+
+const styles = Utils.getStyles(castScreenStyles);
+
+let ds = new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 === r2 });
+
+const dismissButtonSize = 20;
+const castButtonSize = 35;
+
+class CastDevicesScreen extends Component {
+  static propTypes = {
+    height: PropTypes.number.isRequired,
+    width: PropTypes.number.isRequired,
+    onDismiss: PropTypes.func.isRequired,
+    onDeviceSelected: PropTypes.func.isRequired,
+    config: PropTypes.object.isRequired,
+    deviceIds: PropTypes.array.isRequired,
+    deviceNames: PropTypes.array.isRequired,
+    selectedItem: PropTypes.string.isRequired,
+  };
+
+  componentWillMount() {
+    const { height, deviceNames } = this.props;
+
+    this.state = {
```

Comment:
the same as above

Reply:
deleted because of animation isn't used at all

Reply:
still here

Reply:
Sorry for incorrect comment above. We deleted only unused animation with translateY. We left opacity.